pkg/analyzer/analyzers/asana/asana.go

```diff
@@ -59,13 +59,8 @@ func secretInfoToAnalyzerResult(info *SecretInfo) *analyzers.AnalyzerResult {
 		},
 	}
 
-	for _, perm := range permissions {
-		binding := analyzers.Binding{
-			Resource:   userResource,
-			Permission: perm,
-		}
-		result.Bindings = append(result.Bindings, binding)
-	}
+	// bindings to all permissions to resources
+	analyzers.BindAllPermissions(userResource, permissions...)
```

Comment:
`BindAllPermissions` returns a `[]Binding`, so we'll need to save that in the result.

```go
result.Bindings = append(result.Bindings, analyzers.BindAllPermissions(userResource, permissions...)...)
```

... feel free to format or break it up in whichever way you see fit. I admit it's not the most readable as a one-liner 😅

Reply:
** Feeling dumb ** 
Thank you for point it out. Fixed.